engine/lib/behaviors/invalidQuery.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+const Behavior = require("./behavior")
+
+class InvalidQuery extends Behavior {
+    constructor(repository) {
+        super("INVALID-QUERY", undefined)
+        this.repository = repository
+    }
+
+    async execute() {
+        super.execute()
+        const repository = this.repository
```

Comment:
is this line needed? or a common convention?

i think you can remove otherwise.

Reply:
Done!